ecosystem/sep-0045.md

```diff
@@ -0,0 +1,431 @@
+    "AAAAAQAAAAAAAAAA0rDjCmCu2tWgC4nvNxeBkA6AXR61vOlF9kmFcoEQPlUEH887Ol7oawAT5vYAAAAQAAAAAQAAAAEAAAARAAAAAQAAAAIAAAAPAAAACnB1YmxpY19rZXkAAAAAAA0AAAAg0rDjCmCu2tWgC4nvNxeBkA6AXR61vOlF9kmFcoEQPlUAAAAPAAAACXNpZ25hdHVyZQAAAAAAAA0AAABA42dUagEASp9894JchR8zl8qIqSVc5KhuWgxOu1YWDH2nLBlwwQXAMX4h5i1zA5Z86LU84aSPoVQtdzCeJZG2BgAAAAAAAAABqFtYLzwyYhxwv/zkl3ozLGOrmwcX7Z2erX7Yfyiwq8QAAAAPd2ViX2F1dGhfdmVyaWZ5AAAAAAcAAAASAAAAAfDoJWrNj50KrebJUkGjWRvecQYTug7qhuYl5Q29hDS6AAAADgAAAA5sb2NhbGhvc3Q6ODA4MAAAAAAAEgAAAAAAAAAA0rDjCmCu2tWgC4nvNxeBkA6AXR61vOlF9kmFcoEQPlUAAAAOAAAADmxvY2FsaG9zdDo4MDgwAAAAAAABAAAAAQAAAA4AAAAJODkyMTk0NDUxAAAAAAAAAA=="
+  ],
+  "network_passphrase": "Test SDF Network ; September 2015"
+}
+```
+
+You can examine the example challenge transaction in the
+[XDR Viewer](https://lab.stellar.org/xdr/view?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;&xdr$blob=AAAAAQAAAAHw6CVqzY+dCq3myVJBo1kb3nEGE7oO6obmJeUNvYQ0ukNc84Ms0ZvgAAAAAAAAAAEAAAAAAAAAAeA7wfSg10yaQYZDRmQeyqsepsS//Mb0rbMQxRgDoSVdWAAAAD3dlYl9hdXRoX3ZlcmlmeQAAAAAGAAAADgAAADhDRFlPUUpMS1pXSFoyQ1ZONDNFVkVRTkRMRU41NDRJR0NPNUE1MlVHNFlTNktETjVRUTJMVVdLWQAAAA4AAAADMTIzAAAAAA4AAAAcaHR0cDovL2xvY2FsaG9zdDo4MDgwL2MvYXV0aAAAAA4AAAAObG9jYWxob3N0OjgwODAAAAAAAA4AAAALZXhhbXBsZS5jb20AAAAAAQAAAAA=&type=SorobanAuthorizationEntry;;)
+
+##### Error
+
+Every other HTTP status code will be considered an error. For example:
+
+```json
+{
+  "error": "The provided account has requested too many challenges recently. Try again later."
+}
+```
+
+### Token
+
+This endpoint accepts a signed challenge transaction, validates it and responds with a session
+[JSON Web Token](https://jwt.io/) authenticating the account.
+
+The **Client** submits a challenge (that was previously returned by the [`challenge`](#challenge) endpoint) as a HTTP
+POST request to `WEB_AUTH_ENDPOINT_SEP0045` using one of the following formats (both should be equally supported by the
+server):
+
+- Content-Type: `application/x-www-form-urlencoded`, body:
+  `authorization_entries=<XDR (URL-encoded)>,authorization_entries=<XDR (URL-encoded)>`)
+- Content-Type: `application/json`, body: `{"authorization_entries": ["<XDR>", "<XDR>"]}`
+
+To validate the challenge transaction the following steps are performed by the **Server**. If any of the listed steps
+fail, then the authentication request must be rejected — that is, treated by the **Server** as an invalid input.
+
+1. Construct a transaction with a single Invoke Host Function operation using the authorization entries returned by the
+   client;
+1. Simulate the transaction and verify that it succeeds.
```

Comment:
The list seems pretty small compared to SEP-10's list of things to verify.

This should state how to fill out the InvokeHostOp and transaction. Along with the steps the server needs to take to verify the incoming signed values. At the very least it'll be important to check the server value is signed, as well as consistency between the input parameters across all the authorization entries.